examples/rest_api/rest_api.py

```diff
@@ -0,0 +1,134 @@
+import dbinfo
+import requests
+import json
+
+URL = "http://" + dbinfo.DB_HOST + '/api'
+
+
+def dict_query_to_str(dict_query):
+
+    return str(json.dumps(dict_query)).replace("'", '\"')
+
+
+def parse_auth(res):
+
+    res = json.loads(json.loads(res)["json"])
+    print(res)
+
+    session_token = res[0]["Authenticate"]["session_token"]
+    refresh_token = res[0]["Authenticate"]["refresh_token"]
+    return session_token, refresh_token
+
+
+def auth():
+
+    query = [{
+        "Authenticate": {
+            "username": dbinfo.DB_USER,
+            "password": dbinfo.DB_PASS,
+        }
+    }]
+
+    query = dict_query_to_str(query)
+
+    # Authenticate
+    response = requests.post(URL, data = {'json_query': query})
+
+    # print(response.status_code)
+    # print(response.text)
+
+    return parse_auth(response.text)
+
+
+def query_api(query, st):
+
+    query = dict_query_to_str(query)
+    # print(repr(query))
+
+    response = requests.post(URL, headers = {'Authorization': "Bearer " + st},
```

Comment:
These tokens are to prevent MITM, right? Do we plan to have auth against some auth provider for SSO, AD, etc?

Reply:
this is standard https auth. in the future, yes, we would add SSO and other authentication mechanisms.

Reply:
That's what's nice about the token, it allows a very pluggable auth framework.